/home/eliasd/lensing/pyrofit-lensing-analysis/experiments/swyft_unet/scripts
Image has L = 40.
m samples: ['2.1e+10', '1.9e+10']
Creating new store.
Store: Adding 965 new samples to simulator store.
 Creating store Done! 
 Total creating time is 0:00:49! 
 
 
Traceback (most recent call last):
  File "/home/eliasd/lensing/pyrofit-lensing-analysis/experiments/swyft_unet/scripts/train.py", line 103, in <module>
    run()
  File "/home/eliasd/.pyenv/versions/lens-3.9.7/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/home/eliasd/.pyenv/versions/lens-3.9.7/lib/python3.9/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/home/eliasd/.pyenv/versions/lens-3.9.7/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/eliasd/.pyenv/versions/lens-3.9.7/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/home/eliasd/lensing/pyrofit-lensing-analysis/experiments/swyft_unet/scripts/train.py", line 61, in run
    store = swyft.Store.load(path=store_path)
  File "/home/eliasd/lensing/swyft/swyft/store/store.py", line 605, in load
    raise FileNotFoundError(f"There is no directory store at {path}.")
FileNotFoundError: There is no directory store at /nfs/scratch/eliasd/store/store_real_m_1_nsub_2_nsim_1000.zarr.
Traceback (most recent call last):
  File "/home/eliasd/lensing/pyrofit-lensing-analysis/experiments/swyft_unet/scripts/get_pred.py", line 136, in <module>
    run()
  File "/home/eliasd/.pyenv/versions/lens-3.9.7/lib/python3.9/site-packages/click/core.py", line 1137, in __call__
    return self.main(*args, **kwargs)
  File "/home/eliasd/.pyenv/versions/lens-3.9.7/lib/python3.9/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/home/eliasd/.pyenv/versions/lens-3.9.7/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/eliasd/.pyenv/versions/lens-3.9.7/lib/python3.9/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/home/eliasd/lensing/pyrofit-lensing-analysis/experiments/swyft_unet/scripts/get_pred.py", line 67, in run
    store = swyft.Store.load(path=store_path)
  File "/home/eliasd/lensing/swyft/swyft/store/store.py", line 605, in load
    raise FileNotFoundError(f"There is no directory store at {path}.")
FileNotFoundError: There is no directory store at /nfs/scratch/eliasd/store/store_real_m_1_nsub_2_nsim_1000.zarr.
